app/assets/javascripts/jquery.textfill.min.js

```diff
@@ -0,0 +1,20 @@
+/*
+  textfill
+ @name      jquery.textfill.js
+ @author    Russ Painter
+ @author    Yu-Jie Lin
+ @author    Alexandre Dantas
+ @version   0.6.0
+ @date      2014-08-19
+ @copyright (c) 2014 Alexandre Dantas
+ @copyright (c) 2012-2013 Yu-Jie Lin
+ @copyright (c) 2009 Russ Painter
+ @license   MIT License
+ @homepage  https://github.com/jquery-textfill/jquery-textfill
+ @example   http://jquery-textfill.github.io/jquery-textfill/index.html
+*/
+(function(m){m.fn.textfill=function(r){function f(){a.debug&&"undefined"!=typeof console&&"undefined"!=typeof console.debug&&console.debug.apply(console,arguments)}function s(){"undefined"!=typeof console&&"undefined"!=typeof console.warn&&console.warn.apply(console,arguments)}function p(a,b,e,k,n,g){function d(a,b){var c=" / ";a>b?c=" > ":a==b&&(c=" = ");return c}f("[TextFill] "+a+" { font-size: "+b.css("font-size")+",Height: "+b.height()+"px "+d(b.height(),e)+e+"px,Width: "+b.width()+d(b.width(),
+k)+k+",minFontPixels: "+n+"px, maxFontPixels: "+g+"px }")}function q(a,b,e,k,f,g,d,h){for(p(a,b,f,g,d,h);d<h-1;){var l=Math.floor((d+h)/2);b.css("font-size",l);if(e.call(b)<=k){if(d=l,e.call(b)==k)break}else h=l;p(a,b,f,g,d,h)}b.css("font-size",h);e.call(b)<=k&&(d=h,p(a+"* ",b,f,g,d,h));return d}var a=m.extend({debug:!1,maxFontPixels:40,minFontPixels:4,innerTag:"span",widthOnly:!1,success:null,callback:null,fail:null,complete:null,explicitWidth:null,explicitHeight:null,changeLineHeight:!1},r);f("[TextFill] Start Debug");
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected '{' and instead saw 'break'.<br>'}' is not a statement label.<br>Expected '{' and instead saw 'h'.<br>Expected an assignment or function call and instead saw an expression.<br>'a' is already defined.<br>'q' is defined but never used.